pallets/passkey/src/types.rs

```diff
@@ -171,6 +171,9 @@ impl VerifiablePasskeySignature {
 
 		// prepare signing payload which is [authenticator || sha256(client_data_json)]
 		let mut passkey_signature_payload = self.authenticator_data.to_vec();
+		if passkey_signature_payload.len() < 37 {
+			return Err(PasskeyVerificationError::InvalidProof);
```

Comment:
Based on documentation for `authenticator_data` this should be atleast 37 bytes

Reply:
We might want to add a new Error if it doesn't exist. Something like  `PasskeyVerificationError::InvalidAuthenticatorData`